lib/random_assign_utils.rb

```diff
@@ -1,6 +1,111 @@
 # frozen_string_literal: true
 
 class RandomAssignUtils
+  def self.log_error(automation, message)
+    raise("[discourse-automation id=#{automation.id}] #{message}.")
+  end
```

Comment:
Is it logging or raising? 🤔 

Should either be `log_error` and do `Rails.logger.error()` or `raise_error` and do `raise`

Reply:
will do raie error because I want to interrupt script